tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+{
+  "type": "Major",
+  "name": "Classics",
+  "code": "CLA",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-classics",
+    "https://classics.princeton.edu/programs/undergraduate/tracks"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Michael A. Flower",
+      "email": "mflower@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Joshua Billings",
+      "email": "jhbillings@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Tracks",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,

```

Why is the explanation empty?
```suggestion
      "explanation": "Three programs of study are offered within the Department of Classics. The first, Classics, uses knowledge of Greek or Latin (or both languages) as a gateway to the study of the literature, history, and culture of ancient Greece and Rome. The second, Classical Studies, allows for the study of different aspects of a specific period or facet of classical civilization and its impact; it does not initially require knowledge of Greek or Latin. The third, Ancient History, focuses on the history of the ancient world, with special attention to historical method and comparative approaches; it does not initially require knowledge of Greek or Latin.",
```